src/AvatarMember.jsx

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import cx from 'classnames';
 import Avatar from './Avatar';
+import { getIconAsBase64Uri } from './utils/base64';
+
+const NO_PHOTO_SRC = getIconAsBase64Uri('profile');
```

Comment:
apparently it's possible to use external SVGs in `background-image` rules - http://stackoverflow.com/a/9193203

Are there technical blockers for that? It's not a huge deal, but I am curious